README.rst

```diff
@@ -208,6 +208,33 @@ using the ``WATCHMAN_ERROR_CODE`` setting::
 
     WATCHMAN_ERROR_CODE = 500
 
+Logging
+*******
+
+watchman includes log messages using a logger called ``watchman``.
+You can configure this by configuring the ``LOGGING`` section of your Django
+settings file.
+
+Here is a simple example that would log to the console:
+
+    LOGGING = {
+        'version': 1,
+        'disable_existing_loggers': False,
+        'handlers': {
+            'console': {
+                'class': 'logging.StreamHandler',
+            },
+        },
+        'loggers': {
+            'watchman': {
+                'handlers': ['console'],
+                'level': 'DEBUG',
+            },
+        },
+    }
+
+More information is available in the
+[Django documentation](https://docs.djangoproject.com/en/2.0/topics/logging/#configuring-logging).
```

Comment:
I don't think this is the right format for links in .rst files, sorry. :)

Reply:
oops, fixed that